src/modules/countComments.js

```diff
@@ -0,0 +1,5 @@
+export default function countComments() {
+  const commentElements = document.querySelectorAll('.itemComment');
```

Comment:
- [ ] Great job on successfully implementing the project requirements! 👏 Your codebase is clean and well-organized, which is commendable. However, as a suggestion for improvement, I recommend using modern JavaScript `ES6` syntax throughout your codebase.

    - One way to achieve this is by utilizing arrow functions (`=>`) instead of the traditional `function` keyword when declaring your functions. Arrow functions offer a more concise and expressive way of defining functions, which can enhance the readability of your code.

     - By adopting `ES6` syntax, your code will be more in line with current industry standards, making it easier for other developers to understand and maintain your codebase.

Overall, your work is impressive. Happy coding! 😊🚀